core/client/app/controllers/setup.js

```diff
@@ -15,6 +15,11 @@ export default Ember.Controller.extend(ValidationEngine, {
     ghostPaths: Ember.inject.service('ghost-paths'),
     notifications: Ember.inject.service(),
 
+    stepOne: true,
```

Comment:
I think I'd probably take a it step further and not even mess around with keeping track of which step the user is by hand.  We can pull the current route off the application controller or routing service and make the decision that way:

controllers/setup.js:
```javascript
appController: Ember.inject.controller('application'),

showBackLink: Ember.computed('appController.currentRouteName', function () {
    var appController = this.get('appController'),
          currentStep = Ember.get(appController, 'currentRouteName');

    return currentStep === 'setup.two' || currentStep === 'setup.three';
}),

backRoute: Ember.computed('appController.currentRouteName', function () {
    var appController = this.get('appController'),
          currentRoute = Ember.get(appController, 'currentRouteName');

    return currentRoute === 'setup.two' ? 'setup.one' : 'setup.two';
})
```

And then the template can be:
```handlebars
{{#if showBackLink}}
    {{#link-to backRoute}}Back{{/link-to}}
{{/if}}
```